meta-balena-common/recipes-devtools/go/go-1.17.13.inc

```diff
@@ -38,7 +38,6 @@ SRC_URI += "\
     file://CVE-2023-29405.patch \
     file://CVE-2023-29402.patch \
     file://CVE-2023-29400.patch \
-    file://CVE-2023-29406.patch \
```

Comment:
What the reason for removing this patch? Is this an upstream cherry-pick?

Reply:
This is a patch backported from 1.20 it triggers this issue https://github.com/moby/moby/issues/45935
As this is a non-threat in our case, we don't need to backport the fix and we can ignore the patch.

Reply:
let's add it to the commit log for reference @aethernet